apps/judicial-system/backend/src/app/modules/subpoena/guards/policeSubpoenaExists.guard.ts

```diff
@@ -21,7 +21,9 @@ export class PoliceSubpoenaExistsGuard implements CanActivate {
     }
 
     // subpoenaId is the external police document id
-    request.subpoena = await this.subpoenaService.findBySubpoenaId(subpoenaId)
+    request.subpoena = await this.subpoenaService.findByPoliceSubpoenaId(
+      subpoenaId,
```

Comment:
Should we maybe rename this as well to `policeSubpoenaId`?

Reply:
Yes 🙌  and from this I spotted several other params that should be renamed as well